turborepo-tests/integration/tests/no-args.t

```diff
@@ -98,8 +98,8 @@ Make sure exit code is 2 when no args are passed
             Override the filesystem cache directory
         --concurrency <CONCURRENCY>
             Limit the concurrency of task execution. Use 1 for serial (i.e. one-at-a-time) execution
-        --continue
-            Continue execution even if a task exits with an error or non-zero exit code. The default behavior is to bail
+        --continue [<CONTINUE>]
+            Specify which tasks should continue running when an error occurs. Use "none" to cancel all tasks. Use "independent-tasks-only" to continue running independent tasks and cancel dependent ones. Use "all" to continue running all tasks
```

Comment:
Unsure of why the actual diff isn't showing on the prysk diff output, but it's missing the possible values
```suggestion
            Specify which tasks should continue running when an error occurs. Use "none" to cancel all tasks. Use "independent-tasks-only" to continue running independent tasks and cancel dependent ones. Use "all" to continue running all tasks [default: none] [possible values: none, independent-tasks-only, all]
```

Reply:
fixed now... i ran into this issue a few times while iterating on the PR, my hunch is that prysk is treating `[...]` as a character class or something? 🤔